.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,15 +6,28 @@ labels: 'needs: discussion'
 assignees: ''
 
 ---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
 
-Those who maintain this project do so as volunteers and we have no control over their schedule or priorities.
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
 
-This means if the feature you are proposing is something you'd like to see added the best way to achieve that is for you to organise the effort required to build it. This could be evangelising for the feature, paying someone, doing it yourself, etc.
 
-Even so it might never get merged because the trade off between additional happy users and maintenance burden is not favourable.
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect how work is assigned across the maintenance and contributing team. -->
+
+
+### How much time will adding it take?
+<!-- Try to estimate how much work adding this feature will require. This information will affect how work is assigned across the maintenance and contributing team. -->
+
+
+### Who can do this work?
+<!-- What skills are needed? Who can be recruited to add this feature? This information will affect how work is assigned across the maintenance and contributing team. -->
```

Comment:
```suggestion
<!-- What skills are needed? Who can be recruited to add this feature? This information will affect who in the the maintenance and contributing team chooses to work on the feature with you. -->
```

Reply:
I'd take this or go another step further with:

> What skills are needed? Who can be recruited to add this feature? This information will affect who chooses to work on the feature with you.

Reply:
Gotcha - I think I'm trying to have them realise that maintaining software is a task that volunteers do. I guess I'm worried that people don't even _think_ about these tasks, so by explicitly naming them its bringing those considerations to the fore.

I totally agree that "maintainers" and "contributors" (hopefully) overlap! Definitely don't want to imply that they're two totally separate groups of people.

I'm going to leave this thread to see if a few other folks comment, but I'm fine with changing it to "who chooses to work on the feature with you" if there are a few votes in that direction 👾

Reply:
+1 on "who chooses" vs. "assigned"

Reply:
I do like "who chooses" here :+1: